File: Greedy/python/02_뒤집기.py
# fast campus 강의
# https://www.acmicpc.net/problem/1439
# 2

# 문제 : 정답 바뀌는구간
# 0 1 : 0 0
# 01 10 : 1 1
# 101 010 : 1 2
# 1010 : 2 3
# 10101 : 2 4
# 101010 : 3 5
# 바뀌는구간 + 1 // 2

S, result = input(), 0
for i in range(1, len(S)):
    if S[i] != S[i - 1]:
        result += 1

print((result + 1) // 2)

File: Greedy/c++/02_뒤집기.cpp
// fast campus 강의
// https://www.acmicpc.net/problem/1439
// 1

#include <iostream>

using namespace std;

int main()
{
    string s;
    cin >> s;

    int cnt = 0;
    for (int i = 0; i < s.size() - 1; i++) {
        if (s[i] != s[i + 1])
            cnt += 1;
    }
    cout << (cnt + 1) / 2;
}